### PR TITLE
Event to base

### PR DESCRIPTION
**Pull request by gestchild**
## Who is this for?

Designers/Devs
References #2862

## What is it doing for them?

Moving event info into base page, so we can see what needs tweaking and what is missing/needs developing

events

### CODE REVIEW

File: common/services/prismic/events.js [hunk 1]
```diff
@@ -16,35 +16,53 @@ import {
...
-    identifiers: [],
-    interpretations: [],
-    isDropIn: false,
-    series: [],
+    audiences: [], // TODO
+    bookingEnquiryTeam: null, // TODO
+    bookingInformation: asHtml(document.data.bookingInformation),
+    bookingType: null, // TODO
+    cost: document.data.cost,
+    format: document.data.format && parseEventFormat(document.data.format),
```
Comment (jamesgorrie) [inline common/services/prismic/events.js]: As `parseEventFormat` isn't used anywhere else - could we not just do the parsing here, leaving the `parsers` file to just contain things that are used everywhere?

File: common/views/components/BasePage/EventPage.js [hunk 2]
```diff
@@ -5,13 +5,77 @@ import BaseHeader from '../BaseHeader/BaseHeader';
...
 import type {Event} from '../../../model/events';
+import {spacing, font} from '../../../utils/classnames';
+import camelize from '../../../utils/camelize';
+import {formatAndDedupeOnDate, formatAndDedupeOnTime, joinDateStrings} from '../../../utils/format-date';
 
 type Props = {|
   event: Event
 |}
 
+function dateInfo(times) {
```
Comment (jamesgorrie) [inline common/views/components/BasePage/EventPage.js]: Could we use `DateInfo`?

File: common/views/components/BasePage/EventPage.js [hunk 3]
```diff
@@ -5,13 +5,77 @@ import BaseHeader from '../BaseHeader/BaseHeader';
...
+  return (
+    <Fragment>
```
Comment (jamesgorrie) [inline common/views/components/BasePage/EventPage.js]: Do we want to be using these?
https://github.com/wellcometrust/wellcomecollection.org/blob/master/common/views/components/DateRange/DateRange.js

Or creating new ones to make sure we're doing dates across the site consistently?

Comment (gestchild) [reply common/views/components/BasePage/EventPage.js]: Good question, will discuss with @GarethOrmerod

**Pull request merged**